src/design/ButtonBar/index.stories.tsx

```diff
@@ -102,6 +102,35 @@ export const ButtonSizes = () => (
   </div>
 );
 
+export const AsChildButtons = () => (
+  <div className="flex flex-col gap-4">
+    <p>No Click, Hover effects on child components except Anchor</p>
+    <ButtonBar>
+      <Button asChild size="lg">
```

Comment:
Could only one Label be enough here? I was just wondering what the difference of Label1 and Label2 is

Reply:
Yes, both labels are the same, we can remove one of them